.changeset/ninety-pears-wash.md

```diff
@@ -0,0 +1,7 @@
+---
+"@jspsych/plugin-visual-search-circle": major
+---
+
+TODO what changed?
```

Comment:
@becky-gilbert Can you edit this markdown file to quickly say how you modified the visual-search-circle plugin in #2133 and how to update?

Reply:
Done. Is this ok? 5a26663

Reply:
Perfect, thanks!